blog/2022-01-13-instant-apps-browser-extension/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Bringing Felvin Instant Apps To Google Search!
+description: My Internship Project
+slug: felvin-instant-apps-browser-extension
+authors:
+  - N-Shar-ma
+tags: []
+---
+
+Abandoning the comfortable familiarity of Google and changing your default search engine to Felvin can be a big leap to take for some. That's why I made a browser extension to make the switch much more smooth for anyone curious about Felvin! **The Felvin Instant Apps Browser Extension allows you to benefit from the full range of Instant Apps right inside the Google Search Page**, without needing to change search engines, if you're not yet ready to take the leap.
+
+Coming soon to the Chrome Web Store!
```

Comment:
Now we can add the link here https://chrome.google.com/webstore/detail/felvin-instant-apps-brows/dmhgpjahhfannndnaghleelgnpieiljl